support/Environments/urania.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/env sh
+
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+spectre_setup_charm_paths() {
+    # Define Charm paths
+    export CHARM_ROOT=/u/guilara/charm_impi_2/mpi-linux-x86_64-smp
+    export PATH=$PATH:/u/guilara/charm_impi_2/mpi-linux-x86_64-smp/bin
+}
+
+spectre_load_modules() {
+    module load gcc/11
+    module load impi/2021.7
+    module load boost/1.79
+    module load gsl/1.16
+    module load cmake/3.26
+    module load hdf5-serial/1.12.2
+    module load anaconda/3/2021.11
+    module load paraview/5.10
+    # Load Spack environment
+    source /u/guilara/repos/spack/share/spack/setup-env.sh
+    spack env activate env3_spectre_impi
+    # Load python environment
+    source $SPECTRE_HOME/env/bin/activate
```

Comment:
I suggest you put this Python environment in some fixed location so others can use it, e.g. `/u/guilara/envs/spectre_python`. Otherwise everyone has to create a new Py env in every spectre checkout.

Reply:
I created this